lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -52,8 +52,20 @@ module.exports.prototype = {
                 return;
             }
 
+            var prevToken = file.getPrevToken(comment, {
+                includeComments: true
+            });
+
+            if (prevToken.type === 'Line') {
+                return;
+            }
+
+            if (prevToken.type === 'Punctuator' && prevToken.value === '{') {
```

Comment:
I'm thinking that since we haven't rolled this rule out yet, we aren't breaking backwards compatibility. I don't think I've met anyone who would expect this rule to force:

```javascript
if (true) {

  // comment
}
```

While I could totally see it being optional in the future and adding more options, I think it is fine to leave it like this for the initial release.